extensions/everhour/src/views/RecentTaskList.tsx

```diff
@@ -11,7 +11,10 @@ export function RecentTaskList() {
   const [isLoading, setIsLoading] = useState<boolean>(true);
 
   const refreshActiveTimer = async () => {
-    const toast = await showToast(ToastStyle.Animated, "Refreshing tasks");
+    const toast = await showToast({
+      style: Toast.Style.Animated,
+      title: "Refreshing tasks",
```

Comment:
Let's keep Capital Case for consistency like for all other toasts 

```suggestion
      title: "Refreshing Tasks",
```